lib/info.gi

```diff
@@ -314,6 +314,33 @@ BIND_GLOBAL( "InfoDecision", function(selectors, level)
     return ret;
 end );
 
+SHOWN_USED_INFO_CLASSES := [];
```

Comment:
Maybe insert a comment before this block which indicates what the code is used for, both as a hint to somebody trying to understand the code, and to make grepping for `ShowUsedInfoClasses` reveal this location. E.g. like this:
```suggestion
#############################################################################
##
##  The following code is used by ShowUsedInfoClasses.
##
SHOWN_USED_INFO_CLASSES := [];
```